src/test/java/racingcar/wrapper/RaceCountTest.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.wrapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class RaceCountTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = {"1", "10", "10000"})
```

Comment:
```suggestion
    @ValueSource(ints = {1, 10, 10000})
```
`int`형인 `raceCount`를 입력받는 것이라면 `@ValueSource`의 인자로 `int` 형 배열을 사용할 수 있을 것 같아요!
그러면 문자열을 `int`형으로 변환하는 코드가 없어도 테스트가 정상 동작할 것 같습니다.

Reply:
이 부분은 RaceCount 라는 입력값을 래핑하는 클래스입니다. String 으로 받은 후 RaceCount 내부에서 정수로 변환해서 저장하는 작업이 존재해 테스트 입력도 String으로 보냈습니다 !!